packages/ui-informed/src/controls/formControl/FormInfo.tsx

```diff
@@ -0,0 +1,16 @@
+import { cn } from '@utima/ui';
+import { forwardRef, type ComponentProps } from 'react';
+
+export type FormInfoProps = ComponentProps<'span'>;
+
+export const FormInfo = forwardRef<HTMLSpanElement, FormInfoProps>(
```

Comment:
replace forward ref with `memo`, it's not needed in this case (done incorrectly in wue)